app/views/apartments/index.slim

```diff
@@ -0,0 +1,18 @@
+h1 Index Page
+= link_to 'Create new Apartment', new_apartment_path
+- @apartments.each do |apartment|
+  .user-bloc
```

Comment:
why do you use "user-bloc" css class for apparment's description? It's better to use name of classes which describe its content